frontend/app/controllers/restaurant-list.js

```diff
@@ -0,0 +1,124 @@
+import Controller from "@ember/controller";
```

Comment:
proper packaging would be.. controllers/restaurant/list.js etc (dont change now if there is to many job on this)